Resources/config/cache.xml

```diff
@@ -2,15 +2,21 @@
 <container xmlns="http://symfony.com/schema/dic/services"
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <parameters>
+        <parameter key="smartive_handlebars.cache.redis.client">snc_redis.default</parameter>
+        <parameter key="smartive_handlebars.cache.redis.key_prefix">smartive-handlebars:</parameter>
```

Comment:
I actually had it this way but thought it was more "hacky". Changed it back in 2189a55